File: api/code-samples/javascript/klaytnDebug/vMTracing/traceBlockFromFile.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));
    
    const fileName = "/home/sotatek/block.rlp";

    sdk.debug.traceBlockFromFile(fileName, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: api/code-samples/python/klaytnDebug/vMTracing/traceBlockFromFile.py
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

fileName = "/home/sotatek/block.rlp";

sdk = OpenSDK(host)
debug_response = sdk.debug.trace_block_from_file(fileName)

print(debug_response)
